packages/smooth_app/lib/pages/offline_tasks_page.dart

```diff
@@ -0,0 +1,160 @@
+import 'package:flutter/material.dart';
+import 'package:task_manager/task_manager.dart';
+
+// TODO(ashaman999): add the translations later
+class OfflineTask extends StatefulWidget {
+  const OfflineTask({Key? key}) : super(key: key);
+
+  @override
+  State<OfflineTask> createState() => _OfflineTaskState();
+}
+
+class _OfflineTaskState extends State<OfflineTask> {
+  Future<List<Task>> _fetchListItems() async {
+    final Iterable<Task> tasks = await TaskManager().listPendingTasks();
+    return tasks.toList();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Pending Background Tasks'),
+        actions: <Widget>[
+          PopupMenuButton<int>(
+            onSelected: (int item) async {
+              String status = 'All tasks Caneclled';
+              try {
+                await TaskManager().cancelTasks();
+              } catch (e) {
+                status = 'Something went wrong';
+              }
+              setState(() {});
+              final SnackBar snackBar = SnackBar(
+                content: Text(
+                  status,
+                ),
+                duration: const Duration(seconds: 3),
```

Comment:
Please add constants for long/short… durations and check if we can use them in other places

Reply:
Done this as well !